lib/src/presentation/widgets/movie/movie_presentation.dart

```diff
@@ -66,9 +77,21 @@ class MoviePresentation extends StatelessWidget {
                     posterWidth: posterWidth,
                   ),
                 ),
-                MovieInfo(
-                  padGenresCard: pad1,
-                  movie: movie,
+                Provider<GenresBloc>(
+                  create: (_) => GenresBloc(
+                    fetchGenresUseCase: FetchGenresUseCase(
+                      repository: APIRepository(client: Client()),
+                      database: Provider.of<MovieDatabase>(
+                        context,
+                        listen: false,
+                      ),
+                    ),
+                  ),
+                  builder: (BuildContext ctx, _) => MovieInfo(
```

Comment:
You can replace ctx with context as a parameter name

Reply:
Done 🙈